### PR TITLE
Scrubbing known warnings

### PR DESCRIPTION
**Pull request by lianga888**

### CODE REVIEW

File: zipline/finance/risk/cumulative.py [hunk 1]
```diff
@@ -462,16 +462,17 @@ def calculate_beta(self):
...
 
         # Drop nans if there are gaps in the data
         algorithm_returns = self.algorithm_returns.dropna()
         benchmark_returns = \
             self.benchmark_returns.loc[algorithm_returns.index]
 
+        # it doesn't make much sense to calculate beta for less than two days,
+        # so return none.
+        if len(algorithm_returns) < 2:
+            return 0.0
```
Comment (richafrank) [inline zipline/finance/risk/cumulative.py]: Can you explain this change @lianga888 ?

Comment (lianga888) [inline zipline/finance/risk/cumulative.py]: Talked with @ehebert about this earlier. The warning is returned when we try to calculate covariance between the benchmark and algo when there is only one data point of returns. len(self.annualized_mean_returns) <2: return 0.0 is meant to address that case. However, this length is calculated before NaNs are dropped from algorithm_returns and benchmark_returns

Comment (richafrank) [inline zipline/finance/risk/cumulative.py]: Ah, awesome - sounds good.

**Comment by jikamens** [comment]
@lianga888 @richafrank @twiecki @ehebert Is this PR ready to merge? Because the corresponding Qexec PR that references it by commitish has already been merged to master, which means that currently Qexec master points to a Zipline commitish that isn't on its master. That's no good and needs to be cleaned up ASAP.

**Comment by jikamens** [comment]
**BEWARE. I have just force-pushed this branch*** to rebase it against Zipline master.

**Comment by lianga888** [comment]
I had @richafrank look over it a few days ago, so I believe it's good to go

**Comment by lianga888** [comment]
Sorry, I just realized I never merged my corresponding zipline and qcommons PR when I merged my catch warnings PR yesterday

**Comment by jikamens** [comment]
For future reference:

- I would like to see more detailed commit messages, especially when checking code into Zipline, which is a public repository.
- We try to use commit type labels in the summary lines of commits in Zipline. See https://github.com/quantopian/zipline/wiki/Zipline-development-guidelines .

**Pull request merged**